debian/postinst

```diff
@@ -20,8 +20,26 @@ case "$1" in
     db_get kolibri-server/port
     su $KOLIBRI_USER -c "/usr/share/kolibri-server/kolibri_server_setup.py -d $RET"
     echo "include $DAEMON_HOME/nginx.conf;" > /etc/kolibri/nginx.d/099-user.conf
+
+    # Write logrotate configuration if it's not already written
```

Comment:
I don't know if this is needed after the logging has been drastically reduced after only errors are logged to avoid duplication with nginx logging. In any case, when doing this change I'd like to solve the problem of these files being owned by root while being in an user folder. Do you think that could be solved using logrotate?

Reply:
Great, we need to fix the log ownership - will have a look at that!

I hope we can create log rotation and refine it to fit the needs and avoid infinite growth.

Reply:
Adding a `create` directive here.

```
create 0664 $KOLIBRI_USER $KOLIBRI_GROUP
```

Reply:
Alright, this should be good -- there is a wonderful `size` directive that we could add to put a cap on the maximum size of log files, but I have no idea how to come up with a reasonable value, so filing an issue for that.